test/DebugInfo/DebugUnstructuredControlFlow.cl

```diff
@@ -0,0 +1,29 @@
+// RUN: %clang_cc1 -triple spir64-unknown-unknown -cl-std=CL2.0 -O0 -debug-info-kind=standalone -emit-llvm %s -o %t.ll
+// RUN: llvm-as %t.ll -o %t.bc
+// RUN: llvm-spirv %t.bc --spirv-ext=+SPV_INTEL_unstructured_loop_controls -spirv-text -o %t.spt
```

Comment:
Also you can use `-to-text` option to generate `.spt` file from `.spv` file - in this case you don't need to pass `--spriv-ext` option again

Reply:
Thanks, done!